src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const list = document.querySelectorAll('li');
+
+[...list].map((el) => {
```

Comment:
`map()` method creates a new array with the result of calling the specified function for each element of the array.
if you don’t use it anywhere, you don’t need to create a new array additionally.
use `forEach()` when you need a simple loop and actions for each element. 
`map()` is used to get a new array with each element changed.

Reply:
I was thinking about method -forEach- also, but chose -map- ))